content/academy/pro_apify_scraper_development/integrating_webhooks.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Integrating webooks
+description: Learn how to integrate webhooks into your actors. Webhooks are a super powerful tool, and can be used to do almost anything!
+menuWeight: 6.4
+paths:
+    - pro-apify-scraper-development/integrating-webhooks
+---
+
+# [](#setup) Setup
+
+In this lesson we'll be writing a new actor and integrating it with our beloved Amazon scraping actor, but first, let's remove the code which sends ourselves an email so that we don't end up spamming ourselves when testing:
+
+```JavaScript
+// ...
+log.info('Starting the crawl.');
+await crawler.run();
+log.info('Crawl finished.');
+
+// We don't need the code below anymore!
+
+// log.info('Sending dataset link...');
+// const dataset = await Apify.openDataset();
+// const { id } = await dataset.getInfo();
+
+// await Apify.call('apify/send-mail', {
+//     to: 'youremail@gmail.com',
+//     subject: 'Amazon Dataset',
+//     text: `https://api.apify.com/v2/datasets/${id}/items?clean=true&format=json`,
+// });
+
+// log.info('Finished.');
+//...
+```
+
+Next, we'll navigate to the same directory our **demo-actor** folder lives, and run `apify create filter-actor` _(once again, you can name the actor whatever you want, but for this lesson, we'll be calling the new actor **filter-actor**)_. When prompted for which type of boilerplate to start out with, select **Empty**.
+
+![Selecting an empty template to start with]({{@asset pro_apify_scraper_development/images/select-empty.webp}})
```

Comment:
This image looks a lil weird - like there's a ghost on it :D is it intentional?
<img width="674" alt="Screenshot 2022-04-13 at 10 43 00" src="https://user-images.githubusercontent.com/38246758/163136861-117d97b8-4bae-4bd7-962e-e5bf642a13a4.png">

Reply:
That's my terminal haha

Reply:
I will rescreenshot with a new terminal background